src/cljs/netrunner/gameboard.cljs

```diff
@@ -242,10 +242,40 @@
                (get-in @game-state [:runner :rig (keyword (.toLowerCase (:type card)))]))]
     (some #(= (:title %) (:title card)) dest)))
 
+(defn has?
+  "Checks the string property of the card to see if it contains the given value"
+  [card property value]
+  (when-let [p (property card)]
+    (> (.indexOf p value) -1)))
+
+(defn has-subtype?
+  "Checks if the specified subtype is present in the card.
+  Mostly sugar for the has? function."
+  [card subtype]
+  (has? card :subtype subtype))
+
 (defn playable? [{:keys [title side zone cost type uniqueness abilities] :as card}]
   (let [my-side (:side @game-state)
         me (my-side @game-state)]
     (and (= (keyword (.toLowerCase side)) my-side)
+
+         (cond
+
+           (has-subtype? card "Double")
+           (if (>= (:click me) 2) true false)
```

Comment:
Good job!

The thing that kept me from starting this was an installed Starlight Crusade Funding which messes with this. Can either add a hard-check for that or ignore it since it has rotated.

Reply:
I suspect other cards already break the playable? indicator such as Donut Taganes so I think leaving it is ok - and consistent.  And it is rotating ...

Reply:
Agreed. I got too completionist to start working on it 😉

Reply:
you can always improve it - i am happy!